NDTensors/test/test_dense.jl

```diff
@@ -232,6 +232,22 @@ using .NDTensorsTestUtils: devices_list
   end
 
   # Only CPU backend testing
+  @testset "Contract with exotic types" begin
+    # BigFloat is not supported on GPU
+    M = Tensor(BigFloat, (1, 2, 3))
+    fill!(M, BigFloat(randn(Float64)))
+    O = Tensor(M, (2, 3))
+    N = Tensor([BigFloat(1.0)], (1,))
```

Comment:
```suggestion
    a = randn(BigFloat, 2, 3)
    t = Tensor(a, (1, 2, 3))
    m = Tensor(a, (2, 3))
    v = Tensor([one(BigFloat)], (1,))
```
I think this is a bit cleaner/easier to understand.

Reply:
The function `randn(BigFloat, ...)` is not defined in Julia 1.6 so I wrote this as `a = BigFloat.(randn(Float64, 2, 3))`